### ISSUE TITLE
Uncaught #error {:message "No reader function for tag shadow/env.", :data {:type :reader-exceptio...

### ISSUE THREAD

**Issue by jblunck** [mauricioszabo/atom-chlorine#226]
Using the following shadow-cljs.edn:
```
{:nrepl    {:port 8777},
 :dev-http {8280 {:root             "resources/public",
                  :proxy-predicate  manetu.dev-http/proxy-predicate,
                  :push-state/index "provider/index.html",
                  :proxy-url #shadow/env ["GRPC_WEB_URL"
                                          :default
                                          "https://some.org"]}},
```

**Atom**: 1.53.0 x64
**Electron**: 6.1.12
**OS**: "Fedora release
**Thrown From**: [chlorine](https://github.com/mauricioszabo/atom-chlorine) package 0.10.1

### Stack Trace

Uncaught #error {:message "No reader function for tag shadow/env.", :data {:type :reader-exception, :ex-kind :reader-error}}

```
At /home/jblunck/.atom/packages/chlorine/lib/cmds.js:1759

Error: No reader function for tag shadow/env.
    at /packages/chlorine/lib/cmds.js:1131:26)
    at Function.Rp.h (/packages/chlorine/lib/cmds.js:1133:71)
    at Function.Rp.g (/packages/chlorine/lib/cmds.js:1133:30)
    at NW (/packages/chlorine/lib/cmds.js:1692:388)
    at OW (/packages/chlorine/lib/cmds.js:1692:420)
    at EFa (/packages/chlorine/lib/cmds.js:1698:316)
    at GX (/packages/chlorine/lib/cmds.js:1757:323)
    at tGa (/packages/chlorine/lib/cmds.js:1743:361)
    at IX (/packages/chlorine/lib/cmds.js:1747:322)
    at AGa (/packages/chlorine/lib/cmds.js:1748:110)
    at IX (/packages/chlorine/lib/cmds.js:1747:322)
    at AGa (/packages/chlorine/lib/cmds.js:1748:110)
    at IX (/packages/chlorine/lib/cmds.js:1747:322)
    at AGa (/packages/chlorine/lib/cmds.js:1748:110)
    at JX (/packages/chlorine/lib/cmds.js:1759:198)
    at LX (/packages/chlorine/lib/cmds.js:1758:357)
    at LGa (/packages/chlorine/lib/cmds.js:1760:139)
    at Function.SX.a (/packages/chlorine/lib/cmds.js:1769:365)
    at oPa (/packages/chlorine/lib/cmds.js:2905:580)
    at pPa (/packages/chlorine/lib/cmds.js:2906:111)
    at vQa (/packages/chlorine/lib/cmds.js:3020:362)
    at /packages/chlorine/lib/cmds.js:3807:318
    at /packages/chlorine/lib/cmds.js:3831:483)
    at CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/static/<embedded>:11:350265)
    at Object.didConfirmSelection (/usr/share/atom/resources/app/static/<embedded>:11:684369)
    at SelectListView.confirmSelection (/usr/share/atom/resources/app/static/<embedded>:11:856391)
    at HTMLDivElement.core:confirm (/usr/share/atom/resources/app/static/<embedded>:11:851270)
    at CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/static/<embedded>:11:350265)
    at KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app/static/<embedded>:11:1230852)
    at KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app/static/<embedded>:11:1226986)
    at WindowEventHandler.handleDocumentKeyEvent (/usr/share/atom/resources/app/static/<embedded>:11:284919)
```

### Commands

```
     -0:56.8.0 command-palette:toggle (input.hidden-input)
  2x -0:50.6.0 core:backspace (input.hidden-input)
 11x -0:47.9.0 core:move-up (input.hidden-input)
  3x -0:42.2.0 core:move-down (input.hidden-input)
  5x -0:41.4.0 core:move-up (input.hidden-input)
     -0:37.5.0 core:confirm (input.hidden-input)
     -0:37.4.0 chlorine:connect-socket-repl (input.hidden-input)
     -0:30 command-palette:toggle (input.hidden-input)
  3x -0:28.1.0 core:backspace (input.hidden-input)
 11x -0:20.4.0 core:move-down (input.hidden-input)
     -0:17.9.0 core:move-up (input.hidden-input)
     -0:17.6.0 core:confirm (input.hidden-input)
     -0:17.6.0 chlorine:connect-embedded (input.hidden-input)
```

### Non-Core Packages

```
atom-beautify 0.33.4 
atom-ctags 5.1.2 
atom-ide-base 2.1.1 
atom-ide-ui 0.13.0 
atom-ternjs 0.20.0 
atomic-emacs 0.15.0 
autoclose-html 0.23.0 
autocomplete 0.47.0 
autocomplete-clang 0.13.1 
autocomplete-html-entities 0.2.0 
autocomplete-modules 2.3.0 
busy-signal 2.0.1 
chlorine 0.10.1 
go-debug 1.7.0 
go-plus 6.1.0 
go-signature-statusbar 1.2.3 
hyperclick 0.0.0 
ide-python 1.7.1 
ink 0.12.5 
intentions 1.1.5 
js-hyperclick 1.19.0 
language-babel 2.85.0 
language-rpm-spec 0.16.0 
language-styled-jsx 1.0.4 
linter 3.1.1 
linter-clang 4.1.2 
linter-csslint 2.0.6 
linter-eslint 8.5.5 
linter-js-standard 7.0.0 
linter-ui-default 2.2.3 
lisp-paredit 0.8.0 
parinfer 1.24.1 
prettier-atom 0.60.1 
proto-repl 1.4.24 
proto-repl-charts 0.4.1 
react 0.18.0 
script 3.26.0 
symbol-gen 1.3.1 
toggle-quotes 1.1.4 
```

**Comment by mauricioszabo** [comment]
Thanks for reaching. Wow, I was not aware that shadow-cljs.edn can have data readers. I'll fix it :)

**Comment by mauricioszabo** [comment]
Fixed on 0.10.2
